frontend/src/components/admindashboard/addcategroy.js

```diff
@@ -1,46 +1,40 @@
+import React from 'react';
 import { useState, useRef, useEffect } from 'react';
 import { useNavigate } from 'react-router-dom';
 import axios from '../api/api';
 import Cookies from 'js-cookie';
+import { ToastContainer, toast } from 'react-toastify';
+import { useLocation } from 'react-router-dom';
+
 
 const REG_URL = '/categories';
 const Addcategory = () => {
-    const errRef = useRef();
-    const sucRef = useRef();
+    const loc = useLocation();
```

Comment:
```suggestion
    const location = useLocation();
```

Reply:
I think I should delete const loc sir, I just noticed that it is unused.